libc/fuzzing/__support/hashtable_fuzz.cpp

```diff
@@ -9,110 +9,102 @@
 /// Fuzzing test for llvm-libc hashtable implementations.
 ///
 //===----------------------------------------------------------------------===//
-#include "src/__support/CPP/new.h"
-#include "src/__support/CPP/optional.h"
-#include "src/__support/CPP/string.h"
-#include "src/__support/CPP/utility/forward.h"
+#include "include/llvm-libc-types/ENTRY.h"
```

Comment:
Currently in the implementation of `hsearch` and friends, the definition of `ENTRY` is from `<search.h>`, while in `src/__support/HashTable/table.h` its definition is obtained directly from `include/llvm-libc-types/ENTRY.h`.
These types in `search.h` are perfect candidates for proxy headers.
Can you create an issue to add proxy headers for them and redirect all of their usage in `src` and in here also?

Reply:
sure